sogs/routes/users.py

```diff
@@ -70,6 +72,51 @@ def extract_rooms_or_global(req, admin=True):
     return (None, True)
 
 
+@users.get("/user/inbox")
+@auth.user_required
+def get_inbox():
+    """ paginated get messages """
+    offset = None
+    if 'start_at' in request.query:
+        offset = request.query.get("offset", 0, type=int)
+    n = request.query.get("count", 10, type=int)
+    msgs = [msg.info for msg in g.user.inbox(offset=offset, count=n)]
+    return jsonify(msgs)
+
+
+@users.post("/user/<SessionID:sid>/inbox")
+@auth.user_required
+def send_inbox(sid):
+    """ send a message to a recipiant user via their session id """
+    if sid[0:2] == '05' and not g.user.moderator:
```

Comment:
I don't think there's a use case for sending to a non-blinded id; we can just hard-fail for anyone who tries to send to a non-blinded address.  (Session clients shouldn't ever fetch for their true ID anyway).